docs/wiki/PolicyDeepDive.md

```diff
@@ -70,4 +70,169 @@ Please see the following wiki article that explains this process further: [Addin
+
+- Click on the hyperlink for the initiative definition
+- To determine where the initiative is assigned at any scope in the ALZ management group structure start by getting the initiative name
+
+  ![alz-custom-initiative-def-name](media/alz-update-initiative-with-builtin-02.png)
+
+- Since there is no easy way to get the various scopes an initiative is assigned to, got Azure Resource Graph Explorer
+- Ensure that scope for the query is Directory and then execute the following kusto query:
+
+  ```kusto
+    PolicyResources |
+    where kind =~ 'policyassignments' and tostring(properties.policyDefinitionId) =~ '/providers/Microsoft.Management/managementGroups/contoso/providers/Microsoft.Authorization/policySetDefinitions/Deploy-Diagnostics-LogAnalytics'
+    | extend
+        assignmentScope = tostring(properties.scope),
+        assignmmentNotScopes = tostring(properties.notScopes),
+        assignmmentParameters = tostring(properties.parameters)
+    | project assignmentScope,
+        assignmmentNotScopes,
+        assignmmentParameters
+  ```
+
+- The above command will give a result similar to what is shown below
+
+  ![alz-custom-initiative-assignments](media/alz-update-initiative-with-builtin-03.png)
+
+- As can be seen this particular initiative is assigned with only a single parameter at the following levels in the management group structure
+  - Contoso/
+
+> Note
+that the provided example has a simple parameter set. If more complex parameters are assigned to a policy which is to be migrated those should be noted down. In that respect the possibility to download the query results as CSV could be leveraged.
+
+- Switch from Azure Resource Graph Explorer back to the Policy view
+- Change the scope to include the scope described above, and search for the relevant initiative
+
+  ![alz-delete-initiative-assignments](media/alz-update-initiative-with-builtin-04.png)
+
+- For each assignment, click the ellipsis and select Delete Assignment.
+- Once all initiative assignments are deleted, go to the Definitions pane, search for the initiative definition. Once found click the ellipsis and choose Delete Policy Definition
+
+    ![alz-custom-initiative-def-search](media/alz-update-initiative-with-builtin-01.png)
+
+
+
+### Synchronize your local repo with ALZ-Bicep
+
+To get the latest updates from the ALZ-Bicep repo and download to you local repo, do the following.
+- On a client where you have already cloned your version of the ALZ-Bicep repo, start a PowerShell command prompt
+- Navigate to the root of the local version of the repo
+- Verify that you are on the main branch and up-to-date by running the following commands
+
+  ```posh
+  git checkout main
+  git pull
+  ```
+
+- Synchronize your local repo with the ALZ-Bicep repo by running the following
+command
+  ```posh
+  .github/scripts/Invoke-GitHubReleaseFetcher.ps1 -githubRepoUrl "https://github.com/Azure/ALZ-Bicep"
```

Comment:
Do we need to create a doc to support the usage of this script also?

Reply:
I would say so, if nothing else a readme for the script would be minimum.

Reply:
Take a look at this, might be worth copying? https://github.com/azure/bicep-lz-vending/wiki/consumerguide#creating-your-own-github-repository--utilizing-the-invoke-githubreleasefetcherps1-script--gh-release-checkeryml-github-action-workflow